src/transformers/models/superpoint/image_processing_superpoint.py

```diff
@@ -32,9 +32,16 @@
 from ...utils import TensorType, logging, requires_backends
 
 
+if is_torch_available():
+    import torch
+
+    from .modeling_superpoint import SuperPointKeypointDescriptionOutput
```

Comment:
In case it is used only for proper typehints
```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from .modeling_superpoint import SuperPointKeypointDescriptionOutput
```

Reply:
Ah yes you are right, it is indeed only used for typing, changed it